IAmALamp/src/screens/Login/styles.ts

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  bodyWrapper: {
+    marginLeft: 38,
```

Comment:
You can replace `marginLeft` and `marginRight` to `marginHorizontal` since they have the same value.